mycore-base/src/main/resources/xsl/functions/i18n.xsl

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="3.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                xmlns:map="http://www.w3.org/2005/xpath-functions/map"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:mcri18n="http://www.mycore.de/xslt/i18n"
+                exclude-result-prefixes="fn xs">
+
+    <xsl:function name="mcri18n:translate" as="xs:string">
+        <xsl:param name="key" as="xs:string"/>
```

Comment:
we should (at least as a dummy support) a 2nd parameter "values" that is a sequence of string values that can be filled in gaps in the translation. If this translate function is in use it is harder to implement it later.

Reply:
For the replacements?